src/Essentials/samples/Sample.Server.WebAuthenticator/Essentials.Sample.Server.WebAuthenticator.csproj

```diff
@@ -5,11 +5,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AspNet.Security.OAuth.Apple" Version="3.0.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.Facebook" Version="3.1.2" />
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.Google" Version="3.1.2" />
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.MicrosoftAccount" Version="3.1.2" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="3.1.2" />
+    <PackageReference Include="AspNet.Security.OAuth.Apple" Version="6.0.0" />
```

Comment:
This is a community package and is semi-differently versioned. I am getting the package updated in the dotnet-public mirror. I think it will pull in the latest version, or 6.0.4. That might get us a green build.

Reply:
Yeah, I maintain this package 😄

I assumed you'd just want the 6.0.0 baselines, rather than _latest_, but I can update it if needed.

Reply:
This is just used in the sample, right? So it should be OK.

Reply:
I believe so.

Reply:
won't we neeed this mirror on internal feeds? @mattleibow  ?

Reply:
we will, but I think I asked a while ago. We shall see in a few mins.